packages/react/context-menu/src/ContextMenu.stories.tsx

```diff
@@ -454,10 +454,14 @@ export const WithLabels = () => (
 
 export const CheckboxItems = () => {
   const checkboxItems = [
-    { label: 'Bold', state: React.useState(false) },
-    { label: 'Italic', state: React.useState(true) },
-    { label: 'Underline', state: React.useState(false) },
-    { label: 'Strikethrough', state: React.useState(false), disabled: true },
+    { label: 'Bold', state: React.useState<boolean | 'indeterminate'>(false) },
+    { label: 'Italic', state: React.useState<boolean | 'indeterminate'>(true) },
+    { label: 'Underline', state: React.useState<boolean | 'indeterminate'>(false) },
+    {
+      label: 'Strikethrough',
+      state: React.useState<boolean | 'indeterminate'>(false),
+      disabled: true,
+    },
```

Comment:
Can we update this story the same as `DropdownMenu` and `Menu`?

Reply:
Can do, but that will prevent using a disabled checkbox in the story, which was something that was present before my changes. Any concerns about that?

Reply:
Maybe just add an extra disabled static one below after a separator?